packages/react-components/react-search/docs/MIGRATION.md

```diff
@@ -0,0 +1,94 @@
+| multiline mode               | `multiline?: boolean`                                 | separate component                      |
+| underlined                   | `underlined?: boolean`                                | `appearance`                            |
+| borderless                   | `borderless?: boolean`                                | not supported?                          |
+| fluid (full width)           | default behavior                                      | default behavior                        |
+| inline                       | n/a (use styling)                                     | `inline?: boolean`                      |
+| label                        | `label?: string`, `onRenderLabel`                     | handled by Field                        |
+| label position               | n/a (use styling)                                     | handled by Field                        |
+| description                  | `description?: string`, `onRenderDescription`         | handled by Field                        |
+| error message                | see [Spec-variants.md](./Spec-variants.md#validation) | handled by Field                        |
+| content outside before field | `prefix?: string`, `onRenderPrefix`                   |                                         |
+| content outside after field  | `suffix?: string`, `onRenderSuffix`                   |                                         |
+| icon at start of field       | n/a                                                   | `contentBefore` slot                    |
+| icon at end of field         | `iconProps?: IIconProps`                              | `contentAfter` slot                     |
+| value                        | `value?: string`                                      | `value?: string`                        |
+| defaultValue                 | `defaultValue?: string`                               | `defaultValue?: string`                 |
+| onChange                     | `(ev, value) => void`                                 | `(ev, data: { value: string }) => void` |
+| onClear                      | n/a                                                   | `(ev?: any) => void` on `dismiss` slot  |
+| onSearch                     | n/a                                                   | not supported                           |
+| container className          | `className?: string`                                  | top-level `className`                   |
+| input className              | `inputClassName?: string`                             |                                         |
+| aria label                   | `ariaLabel?: string`                                  | `'aria-label'?: string`                 |
+| clearable                    | n/a                                                   | `dismiss` slot                          |
```

Comment:
I think v8 SearchBox just has `clearButtonProps`, `iconProps`, and `showIcon` rather than a `clearable` prop